### PR TITLE
Is malicious indicator found test playbook

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+        - if 'expectedValue' is given: ensure that the given value is equal to the context path
+      scriptName: VerifyContext
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      expectedValue: {}
+      fields: {}
+      path:
+        simple: xxx
```
Comment (asafshen) [inline TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml]: I would change text here to imply which task failed (something like "Found malicious indicator but its not exists..")

File: TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
```
Comment (asafshen) [inline TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml]: I would change text here to imply which task failed (something like "Found malicious indicator but only suspicious  exists..")

File: TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
```
Comment (asafshen) [inline TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml]: I would change text here to imply which task failed (something like "Did not found suspicious indicator but it exists..")

File: TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
```
Comment (asafshen) [inline TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml]: I would change text here to imply which task failed (something like "Did not found malicious indicator but it exists..")

File: Scripts/script-IsMaliciousIndicatorFound.yml [hunk 5]
```diff
@@ -55,3 +59,4 @@ outputs:
 - contextPath: "no"
   description: if did not found any Malicious File/URL/IP in the investigation
 scripttarget: 0
+releaseNotes: 'Fix to Handle includeSuspicious properlly'
```
Comment (Bars92) [inline Scripts/script-IsMaliciousIndicatorFound.yml]: typo `properly`

File: Scripts/script-IsMaliciousIndicatorFound.yml [hunk 6]
```diff
@@ -27,10 +27,14 @@ script: |-
   if (ips && (!Array.isArray(ips) || ips.length > 0)) {
       return 'yes';
   }
-  var indicatorsRes = executeCommand("findIndicators", {"query":"(reputation:Bad or reputation:Suspicious)  and investigationIDs:"+investigation.id,"size":1});
+  var conditionExtra = "";
+  if (args.includeSuspicious == 'yes') {
+      conditionExtra = " or reputation:Suspicious";
+  }
+  var indicatorsRes = executeCommand("findIndicators", {"query":"(reputation:Bad"+conditionExtra+") and investigationIDs:"+investigation.id,"size":1});
```
Comment (Bars92) [inline Scripts/script-IsMaliciousIndicatorFound.yml]: Maybe add extraConditions argument to allow for custom queries like (reputation:Bad or reputation:Suspicious or myCustomField:true)

**Pull request merged**